v3/lint/registration_test.go

```diff
@@ -152,6 +162,116 @@ func TestRegister(t *testing.T) {
 	}
 }
 
+func TestRegistryWithRevocationListLitns(t *testing.T) {
+	expectedNames := []string{
+		"A-mockCertificateLint",
+		"B-mockLint",
+		"C-mockRevocationListLint",
+	}
+
+	expectedSources := []LintSource{
+		Community,
+		RFC3279,
+		RFC8813,
+	}
+
+	egCertificateLint := &CertificateLint{
+		LintMeta: LintMeta{
+			Name:   "A-mockCertificateLint",
+			Source: Community,
+		},
+		Lint: func() CertificateLintInterface { return &mockLint{} },
+	}
+
+	egLint := &Lint{
+		Name:   "B-mockLint",
+		Lint:   func() LintInterface { return &mockLint{} },
+		Source: RFC8813, // arbitrary value for testing
+	}
+
+	egRevocationListLint := &RevocationListLint{
+		LintMeta: LintMeta{
+			Name:   "C-mockRevocationListLint",
+			Source: RFC3279, // arbitrary value for testing
+		},
+		Lint: func() RevocationListLintInterface { return &mockRevocationListLint{} },
+	}
+
+	registry := NewRegistry()
+	if err := registry.register(egLint); err != nil {
+		t.Fatal("registry.register failed")
```

Comment:
nit/optional: I suspect they might want you to throw the error into the message.
https://github.com/zmap/zlint/blob/master/v3/lint/registration_test.go#L165